ssg/build_renumber.py

```diff
@@ -139,8 +141,22 @@ def _link_oval_tree(self):
         indexed_oval_defs = map_elements_to_their_ids(
             self.tree, ".//{0}".format(self._get_checkid_string()))
 
-        drop_oval_checks_extending_non_existing_checks(
-            self.tree, self.oval_groups, indexed_oval_defs)
+        defs_miss = get_oval_checks_extending_non_existing_checks(self.tree, indexed_oval_defs)
+        if defs_miss:
+            if self.tolerate_missing_extending_defs:
```

Comment:
This is never set which means this block is never executed. What are the plans?

Reply:
There is no plan - the code is just prepared for the possibility that missing extending definitions become tolerable.

Reply:
Aha, and can it ever happen that it will be tolerable? What is the scenario in which it would be used?

Reply:
Hard to say - you know how it is with assumptions. There must have been some reason why it is tolerated since it was introduced.

Reply:
But it's you who introduces it so you should be able to explain why you introduce it.

Reply:
I have removed the conserved code in a commit that can be reverted in case that there is an interest to build content that has "definition holes".